irohad/ordering/impl/ordering_service_impl.hpp

```diff
@@ -46,11 +48,11 @@ namespace iroha {
        * @param is_async whether proposals are generated in a separate thread
```

Comment:
Please, update comment

Reply:
It should also say `peer_query_factory` on line 42.